docs/get_started.rst

```diff
@@ -1,17 +1,388 @@
+| Do you want to check Apex code coverage when tests are run?      | If yes, checks Apex code coverage when tests are run.                                                                                                                                                                                                                                                 |
++------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Minimum code coverage percentage                                 | Sets the minimum allowed code coverage percentage for your project.                                                                                                                                                                                                                                   |
++------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+
+
+Verify Project Initialization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Verify project initialization was successful by verifying that ``cumulusci.yml`` exists and has contents.
+
+.. code-block:: console
+
+    $ cat cumulusci.yml
+    project:
+        name: SampleProjectName
+        package:
+            name: SamplePackageName
+            namespace: sampleNamespace
+        .
+        .
+        .
+
+The ``cumulusci.yml`` file configures your project-specific tasks, flows, and customizations.
+
+For more information regarding configuration, check out our `project configuration <#TODO internal ref here>`_ section of the docs.
+You can even add and commit it to your Git repository:
+
+.. code-block:: console
+
+    $ git add cumulusci.yml
+    $ git commit -m "Initialized CumulusCI Configuration"
+
+
+
+Add Your Repo to GitHub
+^^^^^^^^^^^^^^^^^^^^^^^
+
+With your ``cumulusci.yml`` file committed, create a repository on GitHub for your new project and push your changes there.
+There are multiple ways to do this:
+
+* Our `Community Project Development with CumulusCI <https://trailhead.salesforce.com/content/learn/modules/community-project-development-with-cumulusci?trail_id=build-applications-with-cumulusci>`_ module covers GitHub Desktop in the "Set Up the Project" section.
+* If you prefer the command line, GitHub has excellent guides on both `git remote <https://github.com/git-guides/git-remote>`_ and `git push <https://github.com/git-guides/git-push>`_.
```

Comment:
```suggestion
With your ``cumulusci.yml`` file committed, create a repository on GitHub for your new project and push your changes there, using whichever method you prefer.

* Our `Community Project Development with CumulusCI <https://trailhead.salesforce.com/content/learn/modules/community-project-development-with-cumulusci?trail_id=build-applications-with-cumulusci>`_ module covers using GitHub Desktop in the Set Up the Project section.
* If you prefer using the command line, GitHub has excellent guides on both `git remote <https://github.com/git-guides/git-remote>`_ and `git push <https://github.com/git-guides/git-push>`_.
```
Suggested rephrasing to avoid the mushiness of "There are" and a "this" without a specified noun.